configs/catalog.json

```diff
@@ -3,113 +3,350 @@
     {
       "name": "Catalog Administrator",
       "system": true,
-      "version": 4,
+      "version": 5,
       "description": "A catalog administrator roles grants create,read,update, delete and order permissions",
       "access": [
         {
-          "permission": "catalog:portfolios:read"
+          "permission": "catalog:portfolios:read",
+          "resourceDefinitions": [
+            {
+              "attributeFilter": {
+                "key": "scope",
+                "operation": "equal",
+                "value": "admin"
+              }
+            }
+          ]
         },
         {
           "permission": "catalog:portfolios:create"
```

Comment:
@mkanoor why was this one changed?

Reply:
@gmcculloug When you are doing a Create on a Portfolio, there are no specific instances involved like when you do a read, update, delete. So the scope is not needed. The API is just going to be a permission check without the scope.